src/js/common/schemaform/save-in-progress/selectors.js

```diff
@@ -0,0 +1,24 @@
+import { createStructuredSelector } from 'reselect';
+import { getFormData } from '../state/selectors';
+
+export const getSaveInProgressState = createStructuredSelector({
```

Comment:
I see this replaced the previous code that just used `createSelector`. Reading the docs and am a bit confused what creating a selector does in general and why the change to the structured one here?

Reply:
The structured selector is a short hand version of what we were doing before. Instead of making you create an object yourself in the function passed as the last argument, it creates the object for you based on the object you pass in.